src/http/headers.cr

```diff
@@ -258,6 +258,73 @@ struct HTTP::Headers
 
   forward_missing_to @hash
 
+  def if_none_match : Array(String)?
```

Comment:
I don't think this parsing method belongs here.

Reply:
Yeah, I wasn't sure whether this should go here or in `Request`. I'll move it to the latter.

Reply:
This wasn't removed.